uiowa/uiowa
# Updated layout_builder_shortcuts module to 1.0.0 release.

# Updates
* Entity type agnostic behavior - added to non-nodes now.
* Generalized link targeting - no longer relies on a class that, frankly, I'm not sure where it actually comes from.

# How to test

- `blt ds && drush @default.local uli node/1/layout`
- Try double-clicking to edit blocks and ensure it still works.
- Visit https://default.local.drupal.uiowa.edu/admin/structure/block/block-content/manage/uiowa_card/display and enable layout builder for card blocks.
- Visit https://default.local.drupal.uiowa.edu/admin/structure/block/block-content/manage/uiowa_card/display/default/layout and confirm that double-clicking to edit blocks works.
